### DISCUSSION TITLE
Strimzi 0.20 vs strimzi 0.32 what are diffences

### DISCUSSION THREAD

**Discussion by amitmiglani91** [strimzi/strimzi-kafka-operator#8080] [General]
can someone please help me with that differences between them strimzi (0.20 and 0.32). and any video like to update strimzi if its there. idint find thats why.  
Planning to upgrade but why its needed looking for that answer.

**Comment by scholzj** [comment]
0.20 is over 2 years old. So there are many changes. You can find the changes in the CHANGELOG: https://github.com/strimzi/strimzi-kafka-operator/blob/main/CHANGELOG.md and in the releases on GitHub. Our videos are on our YouTube channel.

**Reply by amitmiglani91** [reply]
@scholzj  Thanx for reply. 

Can you Please write full step to upgrade from 0.20 to 0.32. or share specific video where you are doing.  its really big help for me

**Reply by amitmiglani91** [reply]
with Helm repo Please.

**Reply by scholzj** [reply]
No, sorry, but I cannot. It is all documented in our documentation for the different releases. I cannot do _individual_ upgrade guides for every user who did not bother to upgrade for 2,5 years. I have neither time nor energy for that. You have to go through the docs and follow them. In the worst case, you can simply follow them version by version.

**Reply by amitmiglani91** [reply]
No Problem, i will try my best to understand Strimzi, yha i understand that, no problem, believe me if its in my hand i will definitely update Strimzi on time, thanq so much for time & reply.
